app/controllers/api/v0/tasks_controller.rb

```diff
@@ -3,6 +3,7 @@ module V0
     class TasksController < ApplicationController
       include Api::V0::Mixins::IndexMixin
       include Api::V0::Mixins::ShowMixin
+      include Api::V0::Mixins::UpdateMixin
```

Comment:
I think this should only be in v0.1 only.  Also it requires a change to the swagger.yaml

Reply:
Okay, and it is already in the swagger.yaml: https://github.com/ManageIQ/topological_inventory-api/blob/master/public/doc/swagger-2-v0.1.0.yaml#L1407-L1427

Reply:
It's already in the V0x1::TasksController too: https://github.com/ManageIQ/topological_inventory-api/blob/master/app/controllers/api/v0x1.rb#L83

Reply:
Hm I must have been using v0.0 when it was failing before, v0.1 already has the mixin